core/tests/test_utils.py

```diff
@@ -2383,12 +2766,23 @@ def get_json(
         # https://github.com/Pylons/webtest/blob/bf77326420b628c9ea5431432c7e171f88c5d874/webtest/app.py#L1119
         self.assertEqual(json_response.status_int, expected_status_int)
 
-        return self._parse_json_response(json_response, expect_errors)
+        response: Dict[str, Any] = self._parse_json_response(
+            json_response,
+            expect_errors
+        )
```

Comment:
Missing explanation for the `Any`. Ditto below

Reply:
Done!